[2022-06-20 20:31:08,128] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl-processing.transform manual__2022-06-20T20:30:36.458464+00:00 [queued]>
[2022-06-20 20:31:08,225] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl-processing.transform manual__2022-06-20T20:30:36.458464+00:00 [queued]>
[2022-06-20 20:31:08,227] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 20:31:08,239] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-20 20:31:08,245] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 20:31:08,486] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform> on 2022-06-20 20:30:36.458464+00:00
[2022-06-20 20:31:08,521] {standard_task_runner.py:52} INFO - Started process 3631 to run task
[2022-06-20 20:31:08,582] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl-processing', 'transform', 'manual__2022-06-20T20:30:36.458464+00:00', '--job-id', '680', '--raw', '--subdir', 'DAGS_FOLDER/etl_processing.py', '--cfg-path', '/tmp/tmpbnaeoxoo', '--error-file', '/tmp/tmpc6s7wup0']
[2022-06-20 20:31:08,615] {standard_task_runner.py:80} INFO - Job 680: Subtask transform
[2022-06-20 20:31:09,208] {task_command.py:370} INFO - Running <TaskInstance: etl-processing.transform manual__2022-06-20T20:30:36.458464+00:00 [running]> on host fe209149cfc1
[2022-06-20 20:31:10,070] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl-processing
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2022-06-20T20:30:36.458464+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-20T20:30:36.458464+00:00
[2022-06-20 20:31:10,453] {python.py:173} INFO - Done. Returned value was: None
[2022-06-20 20:31:10,651] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl-processing, task_id=transform, execution_date=20220620T203036, start_date=20220620T203108, end_date=20220620T203110
[2022-06-20 20:31:10,913] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-20 20:31:11,514] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
